wpilibj/src/main/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.java

```diff
@@ -37,6 +37,13 @@ default void setVoltage(double outputVolts) {
    */
   double get();
 
+  /**
+   * Common interface for getting the current voltage of the motor controller.
+   *
+   * @return The current controller voltage output. Value is between 0V and 13V.
+   */
+  double getVoltage();
```

Comment:
Make this a default method like setVoltage() so no changes are needed in derived classes.